src/ergw_aaa_ro.erl

```diff
@@ -768,11 +769,9 @@ request_credits(Session, MSCC) ->
     Credits = maps:get(credits, Session, #{}),
     maps:fold(
       fun(RatingGroup, empty, Request) ->
-	      ?LOG(warning, "Ro Charging Key: ~p", [RatingGroup]),
```

Comment:
Maybe keep as info?

Reply:
no, this is one of my infamous debug messages that are tagged with some other level to make them stand out in logs during development.  It should never have been merged like this.

Even having as a debug message does no long make sense (IMHO). If we want debugging on the requested RGs, then it should go into the place where the request is actually generated.

Reply:
ok